include/power_grid_model/component/transformer.hpp

```diff
@@ -49,9 +50,9 @@ class Transformer : public Branch {
           base_i_to_{base_power_3p / u2_rated / sqrt3},
           nominal_ratio_{u1_rated / u2_rated},
           z_grounding_from_{
-              get_z_grounding(transformer_input.r_grounding_from, transformer_input.x_grounding_from, u1_rated)},
+              calculate_z_pu(transformer_input.r_grounding_from, transformer_input.x_grounding_from, u1_rated)},
           z_grounding_to_{
-              get_z_grounding(transformer_input.r_grounding_to, transformer_input.x_grounding_to, u2_rated)} {
+              calculate_z_pu(transformer_input.r_grounding_to, transformer_input.x_grounding_to, u2_rated)} {
         // check on clock
         bool const is_from_wye = winding_from_ == WindingType::wye || winding_from_ == WindingType::wye_n;
```

Comment:
@TonyXiang8787 Is a check on clock required here. Will just the one in 2 winding be enough?